miniapps/solvers/div_free_solver.cpp

```diff
@@ -586,7 +587,7 @@ void DivFreeSolver::Mult(const Vector & x, Vector & y) const
       ch.Clear();
       ch.Start();
 
-      auto M = dynamic_cast<HypreParMatrix&>(ops_.Last()->GetBlock(0, 0));
+      auto M = dynamic_cast<const HypreParMatrix&>(ops_.Last()->GetBlock(0, 0));
```

Comment:
Nit: `auto &M`

Reply:
A good point, it is not related to the change, but should be fixed too 😉